cohttp/src/server_utils.ml

```diff
@@ -0,0 +1,4 @@
+let resolve_local_file ~docroot ~uri =
```

Comment:
Could you rename Cohttp.Server_utils to Cohttp.Path, perhaps? It's more descriptive.

Reply:
Done.